lightning/src/ln/channel.rs

```diff
@@ -2417,7 +2440,7 @@ impl<Signer: Sign> Channel<Signer> {
 				bitcoin_tx.txid
 			};
 			let htlcs_cloned: Vec<_> = commitment_tx.3.iter().map(|htlc| (htlc.0.clone(), htlc.1.map(|h| h.clone()))).collect();
-			(commitment_tx.2, htlcs_cloned, commitment_tx.0, commitment_txid, commitment_tx.1)
+			(commitment_tx.2, htlcs_cloned, commitment_tx.0, commitment_txid, commitment_tx.1, commitment_tx.4, commitment_tx.5)
```

Comment:
I think we can elide the `commitment_tx.4` since it's unused

Reply:
According to my compiler "destructuring assignements are unstable" (E0658) :thinking: